.eslintrc.json

```diff
@@ -0,0 +1,142 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "parserOptions": {
+        "ecmaFeatures": {
+            "experimentalObjectRestSpread": true,
+            "jsx": true
+        },
+        "sourceType": "module"
+    },
+    "rules": {
+        "arrow-spacing": "error",
+        "brace-style": [
+            "error"
+        ],
+        "camelcase": [
+            "error"
+        ],
+        "capitalized-comments": [
+            "error",
+            "always"
+        ],
+        "comma-spacing": [
+            "error",
+            { "before": false, "after": true }
+        ],
+        "comma-style": [
+            "error",
+            "last"
+        ],
+        "computed-property-spacing": [
+            "error",
+            "never"
+        ],
+        "func-call-spacing": [
+            "error",
+            "never"
+        ],
+        "indent": [
+            "error",
+            2,
+            { "SwitchCase": 1 }
+        ],
+        "jsx-quotes": [
+            "error",
+            "prefer-double"
+        ],
+        "key-spacing": [
+            "error",
+            { "beforeColon": false }
+        ],
+        "keyword-spacing": [
+            "error",
+            { "before": true }
+        ],
+        "linebreak-style": [
+            "error",
+            "unix"
+        ],
+        "lines-around-comment": [
+            "error", 
+            {
+                "beforeBlockComment": true,
+                "beforeLineComment": true,
+                "allowBlockStart": true,
+                "allowObjectStart": true,
+                "allowArrayStart": true
+            }
+        ],
+        "lines-around-directive": [
+            "error",
+            "always"
+        ],
+        "max-len": [
+            "error",
+            80
```

Comment:
80 chars is the default and a somewhat consensual value used as the maximum line length. May be too strict for us? Should we do 90 or 100?